index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse Project</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<h1>Hello Microverse!</h1>
```

Comment:
Kindly add your `h1` heading tag inside a `header` element to meet the requirements. 
_**A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.**_